src/Commands.hs

```diff
@@ -456,6 +460,16 @@ commandOS :: CommandCallback
 commandOS ctx _ =
   return (ctx, (Right (XObj (Str os) (Just dummyInfo) (Just StringTy))))
 
+-- | Command for getting the native architecture.
+commandArch :: CommandCallback
+commandArch ctx _ =
+  return (ctx, (Right (XObj (Str arch) (Just dummyInfo) (Just StringTy))))
+
+-- | Command for getting the name of the operating system you're on.
```

Comment:
Isn‘t it for the operating system youre compiling for rather than the one youre on?

Reply:
`target-triple` and its accessors refer to the target. I didn't want to mess with `os` and added `arch` with that name, but they should probably be renamed to `host-arch` and `host-os`. I can do that in this PR if you want, didn't do so because `os` might be used in a bunch of places.

Reply:
Ah, right. I think that‘s fine?